app/controllers/catalog_controller.rb

```diff
@@ -5,6 +5,30 @@ class CatalogController < ApplicationController
 
   include Blacklight::Catalog
 
+  # Destroy a layer if user is admin and layer is found
+  def destroy
```

Comment:
are the other methods all just one word? maybe we could make this like `destroy_layer` or something

Reply:
or I guess maybe `destroy_document` since it's technically not specific to geoblacklight "layers"

Reply:
In controllers, you usually name methods with one word if it's combined and used with a route. Helper functions can have multiple words.

Reply:
got it. that makes sense